state/processor.go

```diff
@@ -98,7 +153,6 @@ func (tp *TransactionProcessor) randomSort(transactions Transactions) Transactio
 	return transactions
```

Comment:
`transactions[i], transactions[swp] = transactions[swp], transactions[i]` works in go :)